models/staging/tpdm/base/base_tpdm__candidate_educator_preparation_program_associations.sql

```diff
@@ -0,0 +1,35 @@
+with candidate_educator_preparation_program_associations as (
+    {{ source_edfi3('candidate_educator_preparation_program_associations') }}
+),
+renamed as (
```

Comment:
could we add a newline between CTEs?

```suggestion
),

renamed as (
```

Reply:
Hi @ChristianMurphy, it is good idea it looks better with new line, but there is not new line in other models in current repository, if we add new line we break general code style. If it is ok, we will add new line

Reply:
Both styles are in the existing code

https://github.com/Unicon/edu_edfi_source/blob/c865d2839664e34670a6d73148b7ca9a1b21babe/models/staging/edfi_3/intermediate/int_ef3__deduped_descriptors.sql#L1-L29